contracts/arbitrators/KlerosLiquidProxy.sol

```diff
@@ -92,6 +92,11 @@ contract KlerosLiquidProxy is Multicall, IKlerosLiquidProxy {
             claimsManager.isManagerOrMediator(msg.sender),
             "Sender cannot mediate"
         );
+        (, address arbitrated, , , , , , bool ruled) = IKlerosLiquid(
+            address(klerosArbitrator)
+        ).disputes(disputeId);
+        require(arbitrated == msg.sender, "Invalid dispute ID");
```

Comment:
Hmm I'm not sure if this will match. Won't `arbitrated` be the `KlerosLiquidProxy` address, and `msg.sender` be the account that made this function call?

Reply:
Oops, it was meant to be `address(this)`